templates/compose/appflowy.yaml

```
@@ -0,0 +1,217 @@
+# documentation: https://docs.appflowy.io/docs/documentation/appflowy-cloud
+# slogan: AppFlowy is the AI collaborative workspace where you achieve more without losing control of your data.
+# tags: appflowy,notion,open-source,productivity,notes,ai,self-hosted
+# logo: svgs/appflowy.svg
+# category: productivity
+# port: 80
+
+services:
+  appflowy_web:
+    environment:
+      - SERVICE_FQDN_APPFLOWYWEB_80
+      - APPFLOWY_BASE_URL=${SERVICE_FQDN_APPFLOWYCLOUD_8000}
+      - APPFLOWY_GOTRUE_BASE_URL=${SERVICE_FQDN_GOTRUE_9999}
+      - APPFLOWY_WS_BASE_URL=${SERVICE_FQDN_APPFLOWYCLOUD_8000}/ws/v2
```

Since the latest update a few days ago, all `SERVICE_FQDN` and `SERVICE_URL` now work in reverse and need to be swapped.
```suggestion
      - SERVICE_URL_APPFLOWYWEB_80
      - APPFLOWY_BASE_URL=${SERVICE_URL_APPFLOWYCLOUD_8000}
      - APPFLOWY_GOTRUE_BASE_URL=${SERVICE_URL_GOTRUE_9999}
      - APPFLOWY_WS_BASE_URL=${SERVICE_URL_APPFLOWYCLOUD_8000}/ws/v2
```

---

fixed